pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/util/UserUtil.java

```diff
@@ -298,6 +324,11 @@ public static List<String> getUserRolesPerRealm(@NonNull String query, @NonNull
         return roles;
     }
 
+    public static boolean checkGPUserAccessForPatch(String realm, String userId, String userIdRequest, Patch patch) {
```

Comment:
Does it make sense to give this a more generic title like 'checkKitShippingAccess'?

Reply:
Ah... yes! i like that more